scripts/tcl_commands/all.tcl

```diff
@@ -107,10 +120,11 @@ proc prep_lefs {args} {
         if { [info exists ::env(METAL_LAYER_NAMES)] } {
             set ::env(TECH_METAL_LAYERS) $::env(METAL_LAYER_NAMES)
         } else {
-            try_catch openroad -python\
+            try_catch $::env(OPENROAD_BIN) -python\
```

Comment:
should it be like this or the other way around? i think i have seen changes that do the opposite

Reply:
like this, definitely not the other way around: either `$::env(OPENROAD_BIN) -python` or `python3` for things that don't need OpenDB.